Cipher.js

```diff
@@ -188,6 +188,46 @@ export class Cipher {
     return JSON.parse(new TextDecoder().decode(data));
   }
 
+  /**
+   * Creates a JOSE Header and encrypted_key for a recipient.
```

Comment:
This needs to be adjusted because it should not just do the above. Also, this function doesn't actually add a recipient to anything -- so we have a naming issue. This function creates a recipient based on some parameters.